src/components/LetterGrid/LetterGrid.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+
+import { useState } from 'react';
+import styles from './letterGrid.module.scss';
+
+const classNames = require('classnames');
+
+export const LetterGrid = () => {
+  const [wordInput, setWordInput] = useState('');
+
+  const handleUserInput = (id: string) => {
+    const chosenLetter = document.getElementById(id);
+    setWordInput((prev) => prev + chosenLetter?.innerHTML);
+  };
+
+  console.log(wordInput);
+
+  return (
+    <div className={styles.gridContainer}>
```

Comment:
This is very redundant. Try to map through the array of letters and instead of giving classnames for positioning, you could try leveraging :nth-child selector. Keep in mind that we need to be able to switch the order of the letters, so this needs to be dynamic.